contracts/ROSCA.sol

```diff
@@ -162,8 +219,14 @@ contract ROSCA {
    * Any excess funds will be withdrawable through withdraw().
    */
   function contribute() payable {
-    if (!members[msg.sender].alive) throw;
+    if (!members[msg.sender].alive || currentRound == 0) throw;
     members[msg.sender].contributed += msg.value;
+    members[msg.sender].goodStanding = members[msg.sender].contributed / contributionSize;
```

Comment:
I think the name goodStanding is a bit misleading - this is the number of rounds a participant has fully contributed to.
However, this is easily computed from members[msg.sender].contributed, so why do we need to store it in another var?